### ISSUE TITLE
Get the url of original unsplash image

### ISSUE THREAD

**Issue by anagri** [fhuel/Unsplash-It-Sketch#1]
How do you get the url of original unsplash image ? 

If there is no mechanism to do it can we add a shortcut or menu item that copies the url to clipboard ?

**Comment by fhuel** [comment]
Hey anagri, 
I think I never needed the url of the original image, but I imagine that it can be useful for some use cases, good idea!

It seems that the API from usplash.it does not return the ID with the method that is used to retrieve the picture, so I'm not sure if this is possible right now, but I'll look into it...

Anyway,having the url copied to the clipboard would work with just one image at a time, but I presume the plugin is used more often to populate several images at once.
What do you think about having the ID (or even the url) for the original images as layer names?

**Comment by anagri** [comment]
I think having the id of the image as layer name would be perfect. It will not be very long, but give sufficient info on how to retrieve the original image. On the plugin's Github readme we can have a small section about this convention. The layer name can be bg_unplash_id or something similar and we can retrieve the original image using http://unplash.com/photos/id.

I came across this idea when I used unplash to populate a background image and it went perfectly with the rest of the colour palette in the app. Now any other image with a different palette would not have given the same feel, so i was looking if there was a way to get the original image id.
